Resources/doc/index.md

```diff
@@ -44,16 +46,36 @@ Publish assets:
 
     php app/console assets:install --symlink web
 
+### Require via NPM
```

Comment:
this is missing an explanation saying that it is optional (it is only needed when wanting to import the translator using modules).

and you should also document that it is very strongly recommended to keep the version in sync between the npm package and the bundle version being used.